.github/workflows/CI.yml

```diff
@@ -376,6 +377,17 @@ jobs:
         with:
           python-version: '3.11'
 
+      - name: Build latest libva
+        timeout-minutes: 5
+        run: |
+          echo "nproc: $(nproc)"
+
+          git clone https://github.com/intel/libva.git
+          cd libva
+          ./autogen.sh --prefix=/usr --libdir=/usr/lib/x86_64-linux-gnu --enable-x11 --without-legacy
+          make -j $(expr $(nproc) - 1)  # use all but one core
+          sudo make install
+
```

Comment:
```suggestion
          git clone https://github.com/intel/libva.git
          cd libva
          ./autogen.sh --prefix=/usr --libdir=/usr/lib/x86_64-linux-gnu --enable-x11 --without-legacy
          make -j $(nproc)
          sudo make install

```

I don't think we'll have any issue with using all the cores, since this is rather quick. I only did that in the sunshine build because the runner was running out of memory and crashing the VM.

Reply:
Yes, I was in doubt too.

It may also be better to use the latest release tarball instead of the latest git commit.

We could also hardcode a specific version in order to make the builds even more reproducible. (It would require some 2 to 3 updates per year.)

I think I'd prefer using the latest released version. That should be good enough. I only used the git version here because it was easiest with the instructions available. And I needed a test build for the original reporter of the issue.

Reply:
I'd prefer latest release as well.

Reply:
You can use gh cli to download the latest release easily. e.g. `gh release download --archive=zip --repo=intel/libva`

https://cli.github.com/manual/gh_release_download

Reply:
I updated the PR. Thanks for the feedback.